src/components/NotFound.tsx

```diff
@@ -3,13 +3,16 @@ import { useNavigate } from 'react-router-dom';
 
 function NotFound() {
   const navigate = useNavigate();
+  const handleButtonClick = () => {
+    navigate('/jobs', { replace: true });
+  };
 
   return (
     <div className="flex h-full flex-col items-center justify-center">
-      <Title order={3}>404</Title>
-      <Text>Page not found!</Text>
-      <Button className="my-4" onClick={() => navigate(',', { replace: true })}>
-        Back Home
+      <Title order={3}>What are you doing here !</Title>
```

Comment:
Thank you for the PR but the app is targeted toward enterprise companies so the tone should be more neutral/serious for informational things.